bitnami/kafka/templates/NOTES.txt

```diff
@@ -72,36 +72,39 @@ You need to configure your Kafka client to access using SASL authentication. To
 
```

Comment:
We should also change "with the contents below"  with sommething like "by executing these commands:"